odkx-src/app-designer-overview.rst

```diff
@@ -70,17 +71,27 @@ File Browser
 ----------------
 The :guilabel:`File Browser` tab provides a view into what will become the application's directory on the phone.
 
-.. image:: /img/app-designer-overview/file-browser.*
-  :alt: The File Browser tab
+.. image:: /img/app-designer-overview/app-file-browser.*
+  :alt: The File Browser tab in the ODK-X Application Designer
 
 .. _app-designer-overview-scan:
 
 ODK-X Scan Form Designer
 --------------------------
 The :guilabel:`Scan Form Designer` tab presents a drag-and-drop editor for mark-sense form creation.
 
-.. image:: /img/app-designer-overview/scan-form-designer.*
-  :alt: The ODK-X Scan Form Designer tab
+.. image:: /img/app-designer-overview/appscan-form-designer.*
+  :alt: The ODK-X Scan Form Designer tab in the ODK-X Application Designer
 
 See :doc:`scan-form-designer-intro` documentation for more information about this tool.
 
+.. _app-designer-overview-qr:
+
+ODK-X QR Code Generator
+--------------------------
+The :guilabel:`QR Code Generator` tab contains the QR Code Generator, a tool that generates a QR Code to log into the synced Server . The Server URL, username and password details are entered into their specified input fields, then a QR Code is generated. 
```

Comment:
some minor proof reading issues here -- server can be lower case and there's an extra space before the period that can be cut.

Reply:
Thanks for reviewing. I'll fix that now.